common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -60,6 +60,33 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
 
   }
 
+  object AmpSoundcloud {
+    def createElement(document: Document, trackId: String): Element = {
+      val soundcloud = document.createElement("amp-soundcloud")
+      soundcloud.attr("data-trackid", trackId)
+      soundcloud.attr("data-visual", "true")
+      soundcloud.attr("height", "300") // height is necessary if data-visual == true
+    }
+
+    def getTrackIdFromUrl(soundcloudUrl: String): String = {
```

Comment:
this should probably return an Option[String] in case it isn't a sound cloud url